examples/bri-3/src/bri/workgroup/workgroups/api/dtos/request/createWorkgroup.dto.ts

```diff
@@ -0,0 +1,24 @@
+import { ArrayNotEmpty, IsNotEmpty } from 'class-validator';
+
+export class CreateWorkgroupDto {
+  @IsNotEmpty()
+  name: string;
+
+  @IsNotEmpty()
+  @ArrayNotEmpty()
+  administratorIds: string[];
+
+  @IsNotEmpty()
+  @ArrayNotEmpty()
+  participantIds: string[];
+
+  @IsNotEmpty()
+  securityPolicy: string;
+
+  @IsNotEmpty()
+  privacyPolicy: string;
+
+  workstepIds: string[];
```

Comment:
these 2 optional?

Reply:
Currently due to the relationships within the Prisma Schema, when a workstep or workflow is created it is created with a workgroupId (beacause both exist in a one-to-many relationship with workgroup).  A workgroup must be created first, then when worksteps and workflows are created they are linked by the workgroup ID provided during creation. This means that the worksteps and workflows are empty  for a particular workgroup at the time of that workgroup's creation.  We could switch over to some alternative system (where worksteps and workflows can be created and unlinked to any workgroup, then later linked). Let me know what you think.